eth-bytecode-db/eth-bytecode-db/justfile

```diff
@@ -0,0 +1,33 @@
+# just is like `make` but better https://github.com/casey/just
+
+# list all recipes
+default:
+    just --list --unsorted
+
+
+start-postgres docker_name="eth-bytecode-postgres" db_name="eth_bytecode_db":
+    # we run it in --rm mode, so all data will be deleted after stopping
+    docker run -p 5432:5432 --name {{docker_name}} -e POSTGRES_PASSWORD=admin --rm -d postgres
```

Comment:
Also, not exactly sure, but what if we make a default port docker container is starting to be different (e.g., 5433). The default Postgres port could be frequently occupied by the native Postgres instance running on a user's machine. By specifying some non-typical port, we may try to avoid such situations